tests/google-cloud-python/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN git clone https://github.com/GoogleCloudPlatform/google-cloud-python.git
 WORKDIR google-cloud-python
 
 # Checkout the latest release.
-RUN git checkout $(git describe --tags)
+# TODO: Enable once run_unit_tests.py is released
```

Comment:
#41 

Do you want to see something like:

```
# TODO(#41): Enable once run_unit_tests.py is released
```

Or just not have that comment at all?